provision/pull-images.sh

```diff
@@ -9,9 +9,9 @@ for img in tgraf/netperf httpd cilium/demo-httpd \
   registry busybox:latest quay.io/coreos/etcd:v3.1.0 \
   digitalwonderland/zookeeper wurstmeister/kafka \
   cilium/kafkaclient2 cilium/starwars \
-  istio/examples-bookinfo-reviews-v1 istio/examples-bookinfo-productpage-v1 \
-  istio/examples-bookinfo-details-v1 istio/examples-bookinfo-reviews-v2:0.2.8 \
-  cilium/cilium:v1.0.0; do
+  istio/examples-bookinfo-ratings-v1:1.6.0 istio/examples-bookinfo-reviews-v2:1.6.0 \
+  istio/examples-bookinfo-details-v1:1.6.0 istio/examples-bookinfo-reviews-v1:1.6.0 \
+  cilium/cilium:v1.0.0 cilium/cilium:latest; do
```

Comment:
I would remove the `cilium/cilium:latest`, it doesn't make sense to have it

Reply:
The reason to add cilium/cilium:latest is because of the base image `cilium/cilium-builder`, if we have cilium:latest we download the latest builder without set the date of the image, etc.. 

Regards